src/xdocs/config_naming.xml

```diff
@@ -2228,7 +2229,8 @@ class MyClass {
         <source>
 &lt;module name=&quot;ParameterName&quot;&gt;
   &lt;property name=&quot;format&quot; value=&quot;^[a-z]([a-z0-9][a-zA-Z0-9]*)?$&quot;/&gt;
-  &lt;property name=&quot;accessModifiers&quot; value=&quot;protected, package, private&quot;/&gt;
+  &lt;property name=&quot;accessModifiers&quot; value=&quot;protected, package,
+            private&quot;/&gt;
```

Comment:
let keep it code as it was, I do not see changes.

Reply:
Done.